src/test/regress/expected/multi_partitioning_utils.out

```diff
@@ -0,0 +1,338 @@
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1620000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1620000;
+-- ===================================================================
+-- create test functions
+-- ===================================================================
+CREATE FUNCTION generate_alter_table_attach_partition_command(regclass)
+    RETURNS text
+    AS 'citus'
+    LANGUAGE C STRICT;
+CREATE FUNCTION generate_partition_information(regclass)
+    RETURNS text
+    AS 'citus'
+    LANGUAGE C STRICT;
+CREATE FUNCTION print_partitions(regclass)
+    RETURNS text
+    AS 'citus'
+    LANGUAGE C STRICT;
+CREATE OR REPLACE FUNCTION detach_and_attach_partition(partition_name regclass, parent_table_name regclass)
+RETURNS void LANGUAGE plpgsql VOLATILE
+AS $function$
+DECLARE
+	attach_partition_command text := '';
+	command_result text := '';
+
+BEGIN
+   -- first generate the command
+   SELECT public.generate_alter_table_attach_partition_command(partition_name) INTO attach_partition_command;
+
+   -- later detach the same partition
+   EXECUTE 'ALTER TABLE ' || parent_table_name || ' DETACH PARTITION ' || partition_name || ';';
```

Comment:
Would it make sense to also write a C function for this? since we will want to be able to detach as well?